src/gp/inc/GPMSAOptions.h

```diff
@@ -100,6 +100,12 @@ class GPMSAOptions
   void set_autoscale_minmax_scenario_parameter(unsigned int i);
 
   //! Do automatic normalization, using minimum and maximum values in
+  //  the supplied data, for output data i.
+  //
+  //  Normalized values will range from 0 to 1.
+  void set_autoscale_minmax_output(unsigned int i);
```

Comment:
Good to have this for completeness, though Brian Williams recommends always using mean/variance scaling.